x/ccv/consumer/keeper/provider_info.go

```diff
@@ -0,0 +1,55 @@
+package keeper
+
+import (
+	ibctm "github.com/cosmos/ibc-go/v7/modules/light-clients/07-tendermint" //nolint:golint
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/interchain-security/v3/x/ccv/consumer/types"
+	ccvtypes "github.com/cosmos/interchain-security/v3/x/ccv/types"
+)
+
+func (k Keeper) GetProviderChainInfo(ctx sdk.Context) (*types.QueryProviderInfoResponse, error) { //nolint:golint
+	//  get the channelID for the channel to the provider.
+	consumerChannelID, found := k.GetProviderChannel(ctx)
```

Comment:
Should this be named `consumerChannelID` or `providerChannelID`?

I believe the `// get the channelID ...` comment could be removed because it doesn't seem to add too much value.